pypackage/cookiecutter.json

```diff
@@ -24,5 +24,11 @@
     "__hdev_project_type": "library",
     "__ignore__": "",
     "__target_dir__": "",
-    "_jinja2_env_vars": {"lstrip_blocks": true, "trim_blocks": true}
+    "_jinja2_env_vars": {"lstrip_blocks": true, "trim_blocks": true},
+    "__parallel_unit_tests": false,
+    "__format_runner_type": "ubuntu-latest",
+    "__lint_runner_type": "ubuntu-latest",
+    "__tests_runner_type": "ubuntu-latest",
+    "__coverage_runner_type": "ubuntu-latest",
+    "__functests_runner_type": "ubuntu-latest"
```

Comment:
I think cookiecutter recently added a `cookiecutter.json` inheritance feature so we don't have to repeat these and other common settings in all three `cookiecutter.json` files, could look into that sometime